apps/subscribe-button/src/overlay/Overlay.vue

```diff
@@ -69,16 +77,18 @@ export default {
     }
   },
   methods: {
-    focusURL() {
-      document.execCommand('selectAll', false, null)
-    },
-    testCopy() {
+    copyLink() {
       let field = document.querySelector('#copy-url-field')
```

Comment:
Calling DOM APIs in a Vue Component is in most cases a bad pattern.

* Please use child component refs if needed: https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements
* For a quick solution: Use data bindings to show/hide the `copy-success` element
* For a long term solution: this is application state and should belong in a reducer, copying content to the clipboard is a side-effect and should be handled independently